translations/en.json

```diff
@@ -986,6 +986,8 @@
     "open": "Open",
     "open-home": "Open...",
     "open-dapp": "Open ÐApp",
+    "open-dapp2": "Open dApp",
```

Comment:
could you elaborate on why we need dapp2 ?

Reply:
Hi, Thanks for the PR review.
According to the design, the action button should have an `Open dApp` label. But currently, only`Open ÐApp` label is available.
![image](https://user-images.githubusercontent.com/17097240/183434878-5321626e-c6e9-4e64-8105-9b7bb5f31c83.png)